cargo-geiger/src/cli.rs

```diff
@@ -91,6 +106,23 @@ pub fn get_workspace(
     Workspace::new(&root, config)
 }
 
+/// Query the installed rust compiler for the default host triple
+fn get_host_target() -> Option<String> {
+    let rustc_output = Command::new("rustc").arg("-vV").output().ok()?;
+    if rustc_output.status.success() {
+        let output_str = String::from_utf8(rustc_output.stdout).ok()?;
+        for line in output_str.lines() {
+            if line.starts_with("host:") {
+                let parts: Vec<_> = line.split(" ").collect();
+                return parts.get(1).map(|s| s.to_string());
+            }
+        }
+        None
+    } else {
+        None
+    }
```

Comment:
I would probably just do:
```suggestion
    }
    None
```

Reply:
Also, is it worth maybe giving a warning or something if this fails?